pkg/sfu/downtrack.go

```diff
@@ -1150,6 +1153,17 @@ func (d *DownTrack) handleRTCP(bytes []byte) {
 	}
 }
 
+func (d *DownTrack) SetConnected() {
+	if !d.connected.Swap(true) {
+		if d.bound.Load() && d.kind == webrtc.RTPCodecTypeVideo {
+			targetLayers := d.forwarder.TargetLayers()
+			if targetLayers != InvalidLayers {
+				d.receiver.SendPLI(targetLayers.Spatial, true)
```

Comment:
so this would bypass the PLI throttle.. that seems like it might be problematic in case of large rooms / or livestreams and a lot of participants are joining at the same time.

is it possible to handle this differently?

Reply:
I think it might be okay to not force and latch on to key frame coming from any previous PLI request. Video starting up a little late may not be a big issue as long as audio starts up promptly. Yes, would be great to have video also start in parallel, but video has other delays like stream allocator/key frame etc.

Reply:
webrtc clients have a pli throttle too, it will not response to pli repeat less than 1s, that's I drop the pli request before downtrack connected